src/core.js

```diff
@@ -624,8 +624,16 @@ $.extend( $.validator, {
 					return i;
 				} ).length,
 				dependencyMismatch = false,
-				val = this.elementValue( element ),
-				result, method, rule;
+				val, result, method, rule;
+
+			// If a normalizer is defined for this element, then
+			// call it to retreive the changed value instead
+			// of using the real one.
+			val = typeof rules.normalizer === "function" ? rules.normalizer.call( this, element ) : this.elementValue( element );
```

Comment:
Yes, please use a if.

Technically it can return anything, but we require a string here?